src/index.html

```diff
@@ -9,6 +9,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container header__content">
+        <a href="#home" 
+          class="header__logo">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" 
+              class="nav__link is-active">Apple</a>
```

Comment:
```suggestion
              <a href="#" class="nav__link is-active">Apple</a>
```
do it in other places